ingest/muxer.go

```diff
@@ -1286,7 +1278,6 @@ loop:
 			}
 			continue
 		}
-		im.Info("Connection to %v established, completing negotiation & requesting approval to ingest", tgt.Address)
```

Comment:
can we make this a LocalInfo

It helps with debugging.

Reply:
he's just moved it lower down